wgsl/index.bs

```diff
@@ -193,6 +193,15 @@ The <dfn noexport>roundUp</dfn> function is defined for positive integers |k| an
 
 * roundUp(|k|, |n|) = &lceil;|n| &div; |k|&rceil; &times; |k|
 
+The <dfn noexport>transpose</dfn> of an |n|-column |m|-row matrix |A| is the |m|-column |n|-row matrix
```

Comment:
should we refer to the builtin method instead of re-defining it here?

Reply:
There is no builtin function for transpose.

Reply:
I'm surprised there isn't one. Transpose should be available everywhere, AFAIK